bindata/network/multus-admission-controller/admission-controller.yaml

```diff
@@ -42,6 +43,8 @@ spec:
       labels:
         app: multus-admission-controller
         namespace: {{.AdmissionControllerNamespace}}
+        hypershift.openshift.io/hosted-control-plane: {{.AdmissionControllerNamespace}}
```

Comment:
These are hypershift specific so they should be under `{{- if .HyperShiftEnabled}}` condition.